build.gradle.kts

```diff
@@ -15,12 +15,13 @@ allprojects {
 
     dependencies {
         implementation("javax.inject:javax.inject:1")
+        implementation("org.springframework.boot:spring-boot-starter-web:2.3.3.RELEASE")
```

Comment:
If you move this implementation maybe you don't need to add this implementation on each module. Do you have check it?

Reply:
not yet, only i need on the adapters, the rest is removed